src/assets/interest.scss

```diff
@@ -0,0 +1,69 @@
+@import 'vars';
+
+.interestForm {
+  font-size: 20px;
+  display: block;
+  margin-top: 40px;
+  input {
+    border: 1px solid $darkBlue-o;
+    font-size: inherit;
+    font-family: $font-body;
+    border-radius: 100px 0 0 100px;
+    background: white;
+  }
+  input:active {
+
+    box-shadow: 0px 0px 15px 15px $lightBlue-o;
+  }
+  button {
+    margin-left: -30px;
+    display: inline;
+    font-size: inherit;
+    color: $white;
+    font-family: $font-title;
+    font-weight: bold;
+    background: $pink;
+    border-radius: 100px;
+    border: 1px solid $pink;
+    border-bottom-width: 3px;
+  }
+  input:focus, input:active, input:hover {
+    outline: none;
+    box-shadow: 0px 0px 10px 10px $lightBlue-o;
+
+  }
+  input, button {
+    padding: 20px;
+  }
+
+}
+
+.con {
```

Comment:
`con` is somewhat of a vague class name, if this is just a one-off class, then maybe something like Homepage-content  (if that's what it's for?) would be more apropos?